generator/features/assetListing.ts

```diff
@@ -134,17 +135,24 @@ export const assetListing: FeatureModule<Listing[]> = {
   build({pool, cfg}) {
     const response: CodeArtifact = {
       code: {
-        constants: cfg
-          .map((cfg) => [
-            `address public constant ${cfg.assetSymbol} = ${translateJsAddressToSol(cfg.asset)};`,
-            `uint256 public constant ${cfg.assetSymbol}_SEED_AMOUNT = 1e${cfg.decimals};`,
-          ])
-          .flat(),
-        execute: cfg.map(
-          (cfg) =>
-            `IERC20(${cfg.assetSymbol}).forceApprove(address(${pool}.POOL), ${cfg.assetSymbol}_SEED_AMOUNT);
-            ${pool}.POOL.supply(${cfg.assetSymbol}, ${cfg.assetSymbol}_SEED_AMOUNT, address(${pool}.COLLECTOR), 0);`,
-        ),
+        constants: cfg.map((cfg) => {
+          let listingConstant = `address public constant ${cfg.assetSymbol} = ${translateJsAddressToSol(cfg.asset)};\n`;
+          listingConstant += `uint256 public constant ${cfg.assetSymbol}_SEED_AMOUNT = 1e${cfg.decimals};\n`;
+          if (isAddress(cfg.admin)) {
+            listingConstant += `address public constant ${cfg.assetSymbol}_ADMIN = ${translateJsAddressToSol(cfg.admin)};\n`;
```

Comment:
```suggestion
            listingConstant += `address public constant ${cfg.assetSymbol}_LM_ADMIN = ${translateJsAddressToSol(cfg.admin)};\n`;
```
Just generic admin might be a bit confusing, maybe LM_ADMIN or EMISSIONS_ADMIN would be more better to understand